hazelcast-client/src/test/java/com/hazelcast/client/ClientOwnershipTest.java

```diff
@@ -0,0 +1,150 @@
+package com.hazelcast.client;
```

Comment:
may be you can add this case: client connected, cluster shutdown or all members shutdown, then start the cluster (or one member) again and check owner.